opm/material/fluidsystems/blackoilpvt/ConstantCompressibilityWaterPvt.hpp

```diff
@@ -209,6 +209,31 @@ class ConstantCompressibilityWaterPvt
         return (1.0 + X*(1.0 + X/2.0))/BwRef;
     }
 
+
+    template <class Evaluation>
+    void inverseBAndMu(Evaluation& bw, Evaluation& muW, unsigned regionIdx,
+                      // const Evaluation& /*temperature*/,
+                      const Evaluation& pressure) const
+                      // const Evaluation& /*Rsw*/,
+                      // const Evaluation& /*saltconcentration*/) const
```

Comment:
Would it not be better if this had the same interface as inverseFormationVolumeFactor(), i.e. passing these parameters even if not used? Otherwise it will not be possible to use this new interface as a replacement.

Reply:
Added both interface. Then one not need to add dummy variables if one do not need them.